embark.el

```diff
@@ -919,27 +937,11 @@ TARGETS is the list of targets."
       (setq-local truncate-lines t)
       (setq-local buffer-read-only t)
       (erase-buffer)
-      (insert
-       (if (eq (car target) 'embark-become)
-           (concat (propertize "Become" 'face 'highlight) "\n")
-         (format "%s on%s '%s'\n"
-                 (propertize "Act" 'face 'highlight)
-                 (if (car target) (format " %s" (car target)) "")
-                 (embark--truncate-target (cdr target)))))
+      (insert target)
       (add-face-text-property (point-min) (point)
                               'embark-verbose-indicator-title 'append)
-      (when other-targets
-        (insert
-         (propertize
-          (format "Shadowed targets at point: %s (%s to cycle)\n"
-                  (string-join
-                   (mapcar (lambda (x)
-                             (symbol-name (car x)))
-                           other-targets)
-                   ", ")
-                  (key-description
-                   (car (where-is-internal #'embark-cycle keymap))))
-          'face 'embark-verbose-indicator-shadowed)))
+      (when (and other-targets (where-is-internal #'embark-cycle keymap))
```

Comment:
Why do you test for the presence of `embark-cycle` too?

Reply:
embark-cycle is not present in the sub keymap anymore. At this point you cannot switch anymore, so it would be confusing to still show this.

Reply:
We could also try to support embark-cycle in subkeymaps though, such that cycling would still be possible. But then we could weird behavior, you would end up at the top keymap again.

Reply:
Oh! I see. Let's leave it like this, you enter a prefix keymap at your own risk.

Reply:
Exactly, and you can always abort.